server/src/main/java/org/opensearch/repositories/RepositoriesService.java

```diff
@@ -498,6 +566,25 @@ public Repository repository(String repositoryName) {
         throw new RepositoryMissingException(repositoryName);
     }
 
+    /**
+     * Returns registered crypto client
+     * @param repositoryMetadata repository metadata for which crypto client needs to be returned.
+     * @return crypto client
+     */
+    public CryptoClient cryptoClient(RepositoryMetadata repositoryMetadata) {
```

Comment:
This method isn't used anywhere.

Reply:
It is being used in other PR. Let me move this over there.